app/src/modules/insights/routes/dashboard.vue

```diff
@@ -244,7 +258,7 @@ export default defineComponent({
 
 		const now = new Date();
 
-		const panels = computed(() => {
+		const panels: ComputedRef<Partial<Panel>[]> = computed(() => {
```

Comment:
Lets use the type override on `computed` directly here:

```suggestion
		const panels = computed<Partial<Panel>[]>(() => {
```